automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala

```diff
@@ -61,34 +75,50 @@ trait GPAllocUtils extends BillingFixtures with LeonardoTestUtils {
                                                       BillingProject.BillingProjectRole.User)(hermioneAuthToken)
       )
       _ <- loggerIO.info(s"Billing project claimed: ${claimedBillingProject.projectName}")
-    } yield GoogleProject(claimedBillingProject.projectName)
+      workspaceName <- IO(UUID.randomUUID().toString)
+      _ <- IO(
+        Orchestration.workspaces.create(claimedBillingProject.projectName, workspaceName)(ronAuthToken)
+      )
+      workspaceDetails <- IO(
+        Rawls.workspaces.getWorkspaceDetails(claimedBillingProject.projectName, workspaceName)(ronAuthToken)
+      )
+      json <- IO.fromEither(parse(workspaceDetails))
+      googleProjectOpt = json.hcursor.downField("workspace").get[String]("googleProjectId").toOption
+      googleProjectId <- IO.fromOption(googleProjectOpt)(
+        new Exception(s"Could not get googleProject from workspace $workspaceName")
+      )
+    } yield GoogleProjectAndWorkspaceName(GoogleProject(googleProjectId),
+                                          WorkspaceName(claimedBillingProject.projectName, workspaceName))
 
   /**
    * Unclaiming billing project claim by Hermione
    */
-  protected def unclaimProject(project: GoogleProject): IO[Unit] =
+  protected def unclaimProject(workspaceName: WorkspaceName): IO[Unit] =
     for {
+      _ <- IO(
+        Orchestration.workspaces.delete(workspaceName.namespace, workspaceName.name)(ronAuthToken)
+      )
```

Comment:
I'm wondering if we should do `.attempt` here (and log if it failed similar to `releaseGPAllocProject`). This way it does not fail the test in `afterAll` if cleanup fails, which can happen if some individual tests fail to delete stuff.